raglweb/__init__.py

```diff
@@ -224,6 +224,7 @@ def games():
 def games_json():
     db = _db_get()
     games = _get_games(db, 'outcomes')
+    games += _get_games(db, 'playoff_outcomes')
```

Comment:
If you could swap these two lines, the JSON result should contain games in the right order (or at least playoffs at the very top instead of at the end of the list)